docs/reference/resource-limits-fees.mdx

```diff
@@ -13,27 +13,13 @@ Resource limitations and fees only apply to smart contract transactions. Read mo
 
 Soroban introduces smart contracts, marking the most transformative upgrade to the Stellar network to date. To maintain network performance, the ecosystem has decided to upgrade the network in a phased approach where network capacity is increased gradually over time. See [this blog post](https://stellar.org/blog/developers/the-stellar-network-s-phased-rollout-of-smart-contracts-the-road-to-mainnet) for more details and [Discord](https://discord.com/invite/zVYdY3ktTn) for current status.
 
-| Network Setting | Phase 0 | Phase 1 (current) | Phase 2 |
-| --- | --- | --- | --- |
-| Soroban Txn per ledger | 1 | 100 | 125 |
-| Max CPU Instructions per Txn | 2.5 million (2M instructions for max Wasm) | 100 million (VM instantiation consumes up to 23M for a 64KB Wasm, and 46M for multiple Wasms of 130KB) | 500 million |
-| Memory limit per Txn | 2 MB | 40 MB | 42 MB |
-| Ledger entry size (including Wasm entries) per Txn | 2 KB | 64 KB | 66 KB |
-| Read/Write Ledger entries per Txn | 3 read; 2 write | 40 read; 25 write | 40 read; 25 write |
-| Read/Write bytes per Txn | 3.2 KB read; 3.2 KB write | 130 KB read; 65 KB write | 133 KB read; 66 KB write |
-| Transaction size | 10 KB | 70 KB | 70 KB |
-| Persistent entry minimal/initial lifetime | 4,096 ledgers (~5.68 hours) | 120 days | 120 days |
-| Temporary entry minimal/initial lifetime | 16 ledgers | 1 day | 1 day |
-| Max ledger entry expiration bump | 61 days | 6 months | N/A |
-| Events+return value size bytes | 200 B | 8 KB | N/A |
-
```

Comment:
I think we'll actually want to keep this table (at least some? most?). Rather, just remove the `Phase 0` and `Phase 1` columns, and just have:

```markdown
| Network Setting | Value |
| --- | --- |
| Soroban Txn per ledger | 100 |
| etc. | etc. |
```